ProjectSidewalk/SidewalkWebpage
# Resolves #1457: Add admin tool for looking up labels

Resolves #1457.

**Changes**:
* Adds a label search tool to the admin interface.

There's a new tab on the menu bar for searching labels.
![image](https://user-images.githubusercontent.com/25534091/53278744-edee9480-36bf-11e9-824c-e8c626145697.png)

You can type in a label id into the search bar, and it'll show the label on the panorama along with some of its associated metadata.
![image](https://user-images.githubusercontent.com/25534091/53278733-d8796a80-36bf-11e9-9c39-09e53528d9d4.png)

**Testing**:
Testing should be fairly straightforward for this! This shouldn't be touching other parts of the admin interface.
* Click around to make sure that nothing else is broken / is performing strangely.
* Enter in a label id from the database -- it should appear on the popup.
